src/state/opamSysInteract.ml

```diff
@@ -358,12 +358,16 @@ let packages_status packages =
 (* Install *)
 
 let install_packages_commands sys_packages =
+  let yes ?(no=[]) yes r =
+    if OpamStd.Config.env_bool "CIDEPEXTYES" = Some true then
```

Comment:
A reason for not going through `OpamStateConfig` ?

Reply:
When I did that, I had in mind to add a new _CI state_ that aggregates all CI options/environment variables. So, I didn't had it in `OpamStateConfig` to minimise the changes.